modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -23,6 +23,9 @@
  *   @license    @GPLv3
  *   @link       https://www.github.com/aces/Loris-Trunk/
  */
+
+require_once "NDB_Form.class.inc";
+require_once "NDB_Menu_Filter_final_radiological_review.class.inc";
```

Comment:
the NDB_Form isn't needed because it's autoloaded from php/librares, and the menu flter should have __DIR__  at the start to make sure it's explicitly loading the right one.